system/Debug/Toolbar.php

```diff
@@ -395,8 +395,9 @@ public function prepare(RequestInterface $request = null, ResponseInterface $res
 
 			if (strpos($response->getBody(), '<head>') !== false)
 			{
+				$count = 1;
```

Comment:
I'm pretty sure this variable is unused. Do we need this somewhere else?

Reply:
Ayup. Leftover from my debacle above. I will remove next time I'm on desktop.